src/assets/scss/components/statusContainer.scss

```diff
@@ -0,0 +1,106 @@
+.status-container {
+  background-color: $off-white;
+  border-radius: 5px;
+  box-shadow: 0px 0px 1px 2px rgba(85, 3, 79, 0.1);
+  display: inline-block;
+  margin-right: 10px;
+  min-height: 500px;
+  min-width: 300px;
+
+  &-box {
+    flex-wrap: nowrap;
+    overflow: auto;
+    padding: 10px 5px;
+  }
+
+  .status {
+    margin: 5px auto;
+    padding: 3px 16px;
+    text-align: left;
+  }
+
+  .wrapper {
+    margin-top: 5px;
+    max-height: 600px;
+    overflow: auto;
+
+    &::-webkit-scrollbar {
+      width: 5px;
+    }
+
+    &::-webkit-scrollbar-track {
+      width: 15px;
+    }
+
+    &::-webkit-scrollbar-thumb {
+      background: $light-gray;
+    }
+  }
+
+  .story {
+    align-items: baseline;
+    border-radius: 10px;
+    flex-direction: column;
+    height: fit-content;
+    margin: 5px auto 10px auto;
+    padding: 10px;
+    width: 90%;

```

Comment:
we can remove this with 90% and instead change the margin to be always 10%. This will be more consistent as using witdh is very unpredictable. On larger screens it will not behave as expected and there will be more empty space around. Margin is always a guarantee

Reply:
Let's also remove the shadow when the story is inside the status-container and add a thin border.

so.. this is what the CSS should look like:

```
    align-items: baseline;
    border-radius: 10px;
    flex-direction: column;
    height: -moz-fit-content;
    height: fit-content;
    margin: $eos-s;
    padding: $eos-s;    
    box-shadow: none;
    border: 1px solid $eos-gray-100;
```

notice that I used the spacing vars `$eos-s` instead of plain numbers for pixels, as this is better for maintenance and UI consistency too